src/main/java/com/srans/nestserver/service/VacateService.java

```diff
@@ -129,38 +124,36 @@ public Vacation processVacation(Vacation vacate) throws MailException, Messaging
 	Map<String, Object> reqParamtersMap;
 
 	public String getTemplate(String templateFileName, User user) {
-		logger.trace("In::");
+		logger.trace("In:: getTemplate" + templateFileName);
 		reqParamtersMap = new HashMap<>();
 
 		System.out.println(user.getUserId());
 		reqParamtersMap.put("name", user.getName());
-		Long hostelid=tenantBookRepo.findHostelId(user.getUserId());
-		
-		tenantBookRepo.findAll().forEach(tenantInfo ->{
-			if(tenantInfo.getTenantId()==user.getUserId()) {
-				 floorId=tenantInfo.getFloorId();
-				 roomId=tenantInfo.getRoomId();
-				 roomRent=tenantInfo.getRoomRent();
-				 roomBedId=tenantInfo.getRoomBedId();
-				 
+		Long hostelid = tenantBookRepo.findHostelId(user.getUserId());
```

Comment:
Make sure you follow java standards camelcase

Reply:
Yes sir, I'll follow